### ISSUE TITLE
brew install fails

### ISSUE THREAD

**Issue by prokhor-andrew** [appsquickly/TyphoonSwift#8]
While trying to install TyphoonSwift via 
`brew install appsquickly/core/typhoon`

it shows lots of error-messages: 
`Error: Invalid formula: path/is/here`

and terminates with error:
`Error: Cannot tap appsquickly/core: invalid syntax in tap!`

This happens on Xcode 9.4.1
and MacOS 10.13.5

**Comment by alexgarbarev** [comment]
Yep, it looks like something was changed (swift version, xcode and maybe more), but not updated. Someone need to update the repo to the latest swift version and environment.
